tests/e2e/expected_results/micronaut-core/micronaut-core.json

```diff
@@ -20,1179 +20,1179 @@
                         "predicateType": "https://slsa.dev/provenance/v0.2",
                         "subject": [
                             {
-                                "name": "build/repo/io/micronaut/micronaut-aop/3.8.7/micronaut-aop-3.8.7-javadoc.jar",
+                                "name": "build/repo/io/micronaut/micronaut-aop/3.9.3/micronaut-aop-3.9.3-javadoc.jar",
```

Comment:
I believe this change is because we are getting the latest release instead of getting the same artifact for the commit we are analyzing. Is that correct?

Reply:
Yes that's right.